helix-view/src/editor.rs

```diff
@@ -274,6 +274,8 @@ pub struct Config {
     /// Whether to color modes with different colors. Defaults to `false`.
     pub color_modes: bool,
     pub soft_wrap: SoftWrap,
+    /// Whether to check for external modifications upon saving. Defaults to `true`.
+    pub prevent_external_modifications: bool,
```

Comment:
This setting needs to be added to the book in `book/src/configuration.md`. Ideally that documentation should explicitly note that this setting may need to be disabled in the rare case that a system has unreliable `mtime`

Reply:
done, let me know if the wording is ok

Reply:
sounds good to me